__tests__/app.test.js

```diff
@@ -63,3 +62,42 @@ it('responds with 404 error if not path is found', ()=>{
     })
   })
 })
+
+describe.only('POST', () => {
+  test('post new comment to article and responds with the new comment posted', () => {
+
+    const newComments = {
+      username: "icellusedkars",
+      body: "nice work icellusedkars"
+    }
+    return request(app).post('/api/articles/4/comments').send(newComments).expect(201).then((response) => {
+      expect(response.body.comment).toMatchObject({
+        comment_id: 19,
+          author: "icellusedkars",
+          article_id: 4,
+          votes: 0,
+          body: "nice work icellusedkars"
+      })
+    })
+  })
```

Comment:
Status 404 User not found is missing {
        comment_id: 19,
          author: "iceeeeeeeeeeee",
          article_id: 4,
          votes: 0,
          body: "nice work icellusedkars"
      }